lib/web/fetch/util.js

```diff
@@ -1055,6 +1055,11 @@ async function fullyReadBody (body, processBody, processBodyError) {
   //    with taskDestination.
   const errorSteps = processBodyError
 
+  if (typeof body.source === 'string') {
+    successSteps(new TextEncoder().encode(body.source))
```

Comment:
I did not check your code and if this makes sense. But I think this wil break without-intl builds . check how we implemented utf8Decode and look if you can implement something for utf8Encode?!

Reply:
thank you very much for your suggestion I will try this